design/one-pager-default-resource-class.md

```diff
@@ -88,66 +116,237 @@ spec:
   engineVersion: "9.6"
 ```
 
-Internally, Crossplane will first check to see if a resource class is referenced. If not, it will check to see if a class annotated as `default` has been created for the given kind. Ultimately, if one does not exist, it will fail to provision the resource.
+Internally, Crossplane would first check to see if a resource class is
+referenced. If not, it checked to see if a class annotated as `default` had been
+created for the given kind. Ultimately, if one did not exist, it failed to
+provision the resource.
+
+## Default Class Reference v1
+
+With the implementation of [strongly-typed resource
+classes](./one-pager-strongly-typed-class.md), the generic `ResourceClass`
+became obsolete and the `Policy` kind was introduced. Each claim kind had a
+corresponding policy kind (i.e. `MySQLInstancePolicy` for `MySQLInstance`,
+etc.). Claims controllers could no longer simply omit a `classRef` because their
+controllers would not know what `kind` they intended to bind to. While the
+`MySQLInstance` claim controller previously knew to look for objects of type
+`ResourceClass` that specified
+`mysqlinstance.database.crossplane.io/default:true`, there were now many
+different class `kinds` that the claim could potentially reference (i.e. GCP
+`CloudsqlInstanceClass`, AWS `RDSInstanceClass`, etc.). `Policies` were
```

Comment:
I should probably change "there were now many" to just "there were many". I took a very different approach with this design doc in that I tried to preserve historical thinking because I believe it informs our current direction. I think the verdict is still out on if this is a good way to propose updated designs though.